src/NfcService.cpp

```diff
@@ -630,6 +630,10 @@ void NfcService::HandleEnableResponse(NfcEvent* aEvent)
     }
   } else {
     code = DisableNfc();
+    if (code == NFC_SUCCESS && mP2pLinkManager->IsLlcpActive()) {
```

Comment:
Yes, you are right.
Originally I planed to create another bug for it but i have not yet created it.
Do you want me to fix 'tag case' in this bug?